ha/util/message_bus.py

```diff
@@ -239,12 +242,21 @@ def register(message_type: str, partitions: int = 1):
             partitions (int): Number of partition.
         """
         admin = MessageBusAdmin(admin_id=MessageBus.ADMIN_ID)
-        try:
-            if message_type not in admin.list_message_types():
-                admin.register_message_type(message_types=[message_type], partitions=partitions)
-        except Exception as e:
-            if "TOPIC_ALREADY_EXISTS" not in str(e):
-                raise(e)
+        for count in range(MESSAGE_BUS_RETRY_COUNT):
```

Comment:
why not try infinite times until the topic gets created eventually? this will avoid any variability in interval and number of retries with respect to larger setups.

Reply:
Infinite waiting/retrying creates a problem of POD hung. That is why we introduced the timeout(https://github.com/Seagate/cortx-utils/pull/836/files). After a certain retry, it is necessary to error out so that POD will be restarted.